vsintegration/src/vs/FsPkgs/FSharp.Project/FS/AppConfigHelper.fs

```diff
@@ -353,13 +353,13 @@ namespace Microsoft.VisualStudio.FSharp.ProjectSystem
                         //TargetMin, targetMax, redirects
```

Comment:
I think this table needs to look more like this:
All of the redirects are compatible with the range of assemblies specified by mintarget, maxtarget and so the app config needs a binding redirect for each listed redirect.

````
//TargetMin, targetMax,   redirects
"2.3.0.0",   "2.3.0.0",   ["2.0.0.0";   "2.3.0.0"]
"2.3.5.1",   "2.3.5.1",   ["2.3.5.0";   "2.3.5.1"]
"3.7.4.0",   "3.7.41.0",  ["3.3.1.0";   "3.7.4.0"]
"3.7.4.1",   "3.7.4.1",   ["3.7.41.0"]
"3.47.4.0",  "3.47.41.0", ["2.3.5.0";   "2.3.5.1";   "3.47.4.0"]
"3.47.41.0", "3.47.41.0", ["3.47.41.0"]
"3.78.4.0",  "3.78.41.0", ["3.78.3.1";  "3.78.4.0"]
"3.78.41.0", "3.78.41.0", ["3.78.41.0"]
"3.259.4.0", "3.259.41.0", ["3.259.3.1"; "3.259.4.0"]
"3.259.41.0","3.259.41.0",["3.259.41.0"]
"4.3.0.0",   "4.4.1.0",   ["2.0.0.0";   "2.3.0.0";   "2.3.5.0";   "4.0.0.0";    "4.3.0.0"]
"4.3.1.0",   "4.4.1.0",   ["3.3.1.0";   "2.3.5.1";   "3.78.3.1";  "3.259.3.1";  "4.3.1.0"]
"4.4.0.0",   "4.4.1.0",   ["3.47.4.0";  "3.78.4.0";  "3.259.4.0"; "4.4.0.0"]
"4.4.1.0",   "4.4.1.0",   ["3.47.41.0"; "3.78.41.0"; "3.259.41.0"; "4.4.0.0"; "4.4.1.0"]
````